### PR TITLE
initial streamlit mock with servers list functionality

### LINKED ISSUE #1: Create streamlit mock

### PR DESCRIPTION
**Pull request by Yousifff**
initial streamlit mock with servers list functionality

### CODE REVIEW

**Comment by Zaubeerer** [comment]
Hey @Yousifff, great job that you already submitted your first PR!

**Comment by Zaubeerer** [comment]
Unfortunately, you committed the `.venv` folder, which usually is ignored via a `.gitignore` file.

We can fix this together in our next session.

For now, just continue and create the `app.py` file in which the streamlit mock code will be, as I didn't see it in the Changes yet.

**Comment by Yousifff** [comment]
App won't run if I move app.py outside frontend folder, streamlit raise an exception that it should be relative path. 
I look into it and couldn't find config.toml to to fix it as bellow link.
https://docs.streamlit.io/library/advanced-features/multipage-apps/custom-navigation

If they're within the same folder it works fine

**Comment by Yousifff** [comment]
Ablle to run form page and switch to servers page from app.py

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.venv
```
Comment (Zaubeerer) [inline .gitignore:1]: great addition of the gitignore file!

File: app.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from st_pages import Page, Section, show_pages, add_page_title
+from frontend.form import create_form
```
Comment (Zaubeerer) [inline app.py:2]: I think I know what problem you are referring to regarding relative imports and how to solve it.

We can look into it during our session, but we may as well first focus on other topics

File: frontend/form.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        username = st.text_input("username")
+        password = st.text_input("password", type="password")
+        submit_button = st.form_submit_button("Click")
+        if submit_button:
+            if username and password:
+
+                st.success("Logged in successfully")
+                st.switch_page("frontend/pages/servers.py")
+            else:
+                st.warning("Invalidd username or password")
```
Comment (Zaubeerer) [inline frontend/form.py:17]: Ah, interesting!

Haven't seen login handling in streamlit like this yet, looking forward to running it later.

Comment (Zaubeerer) [inline frontend/form.py:17]: Ah, interesting!

Haven't seen login handling in streamlit like this yet, looking forward to running it later.

File: frontend/pages/servers.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import streamlit as st
+
+
+def servers_list():
+    servers = [f'192.168.100.{str(i)}' for i in range(10)]
+    st.table(servers)
```
Comment (Zaubeerer) [inline frontend/pages/servers.py:6]: nice, simplistic mock 👍

**Review by Zaubeerer — APPROVED** [review]
If it runs, you can merge it and focus on the next issue 👍

**Pull request merged**